src/styles/less/Hud.less

```diff
@@ -31,7 +37,11 @@
 .ctimer.pos-2 {
     margin-left: 33.33%;
     width: 50%;
+<<<<<<< HEAD
```

Comment:
looks like more conflicts with merging. lines should be cleared.

Reply:
thank you for pointing this out, resolved.